salt/postgres/backup.sls

```diff
@@ -0,0 +1,32 @@
+include:
+  - postgres
+
+# Require official PostgreSQL repo as they provide a newer version of pgbackrest.
+pgbackrest:
+  pkg.installed:
+    - name: pgbackrest
+  require:
+    - sls: postgresql
+
+/etc/pgbackrest/pgbackrest.conf:
+  file.managed:
+   - makedirs: True
+   - user: postgres
+   - group: postgres
+   - source: salt://postgres/files/pgbackrest/{{ pillar.postgres.backup.config_name }}.conf
+   - template: jinja
+
+{%- if salt['pillar.get']('postgres:backup:enabled','') == True %}
```

Comment:
The following is sufficient:

```suggestion
{%- if salt['pillar.get']('postgres:backup:enabled') %}
```

Reply:
I have made this change, I also renamed this variable to show it's actual purpose (now called `cron_enabled`).